### PR TITLE
Fix chkpii errors in ssl messages file

### PR DESCRIPTION
**Pull request by pnickoll**

### CODE REVIEW

**Comment by pnickoll** [comment]
#build

**Comment by pnickoll** [comment]
@acdemyers I believe you added this message before, could you buddy check the chkpii fix to ensure you are happy with it pls

File: dev/com.ibm.ws.ssl/resources/com/ibm/ws/ssl/resources/ssl.nlsprops [hunk 1]
```diff
@@ -629,7 +629,7 @@ ssl.client.handshake.error.CWPKI0823E=CWPKI0823E: SSL HANDSHAKE FAILURE:  A sign
 ssl.client.handshake.error.CWPKI0823E.explanation=An error occurred during the SSL handshake.  It might require a signer export/import from the target host to the client TrustStore.
 ssl.client.handshake.error.CWPKI0823E.useraction=Review the extended error message from the TrustManager to determine what needs to change between the target SSL configuration and the client SSL configuration.
 # -------------------------------------------------------------------------------------------------
-ssl.client.handshake.error.CWPKI0824E=CWPKI0824E: SSL HANDSHAKE FAILURE:  Host name verification error while connecting to host [{0}].  The host name used to access the server does not match the server certificate's SubjectDN or Subject Alternative Name information.  The extended error message from the SSL handshake exception is: [{1}].
+ssl.client.handshake.error.CWPKI0824E=CWPKI0824E: SSL HANDSHAKE FAILURE:  Host name verification error while connecting to host [{0}].  The host name used to access the server does not match the server certificate''s SubjectDN or Subject Alternative Name information.  The extended error message from the SSL handshake exception is: [{1}].
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.ssl/resources/com/ibm/ws/ssl/resources/ssl.nlsprops]: Change
"Host name verification error while connecting to host [{0}].  The host name used to access the server does not match the server certificate''s SubjectDN or Subject Alternative Name information.  The extended error message from the SSL handshake exception is: [{1}]."
to
Host name verification error while connecting to host [{0}]. The host name used to access the server does not match the server certificate's SubjectDN or Subject Alternative Name information. The extended error message from the SSL handshake exception is: [{1}]."

**Comment by pnickoll** [comment]
@acdemyers Tagging you in case you want to make changes based on the feedback above. This PR is just for fixing the chkpii error in the NLS files, can you use a different PR if you want to update the messages themselves?

**Pull request merged**